erpnext/regional/saudi_arabia/utils.py

```diff
@@ -28,24 +31,76 @@ def create_qr_code(doc, method):
 
 	for field in meta.get_image_fields():
 		if field.fieldname == 'qr_code':
-			from urllib.parse import urlencode
+			''' TLV conversion for
+			1. Seller's Name
+			2. VAT Number
+			3. Time Stamp
+			4. Invoice Amount
+			5. VAT Amount
+			'''
+			tlv_array = []
+			# Sellers Name
+
+			'''TODO: Fix arabic conversion'''
+			# seller_name = frappe.db.get_value(
+			# 	'Company',
+			# 	doc.company,
+			# 	'company_name_in_arabic')
+
+			# if not seller_name:
+			# 	frappe.throw(_('Arabic name missing for {} in the company document'.format(doc.company)))
+
+			seller_name = doc.company
+			tag = bytes([1]).hex()
+			length = bytes([len(seller_name)]).hex()
+			value = seller_name.encode('utf-8').hex()
```

Comment:
@deepeshgarg007 This is done as per the mentioned specification in the docs but somehow does not convert Arabic name to proper hex code.

Reply:
@ahmadpak I think need to encode string before length function. 
			length = bytes([len(seller_name.encode('utf-8'))]).hex()

Reply:
@ahmadpak I think needs to encode string before length.
			length = bytes([len(seller_name.encode('utf-8'))]).hex()